gnucash/import-export/aqb/gnc-ab-utils.c

```diff
@@ -1008,7 +1008,7 @@ bal_accountinfo_cb (AB_IMEXPORTER_ACCOUNTINFO *element, gpointer user_data)
                        "support Balance download in this Online Banking version. "
                        "In the latter case you should choose a different "
                        "Online Banking version number in the Online Banking "
-                       "(AqBanking or HBCI) Setup. After that, try again to "
+                       "(AqBanking or FinTS) Setup. After that, try again to "
```

Comment:
There's no separate FinTS setup, it's all AQBanking. It's accessed via a button on the Online Banking assistant.

There's also no "Online Banking Version" in AQBanking's setup wizard. There's HBCI Version and RCH Version. Which one affects balance download and is this even germane any more?

Reply:
Right, here is indeed meant HBCI-Version. See image at [step 17 in Onlinebanking-Einrichtungsassistent](https://wiki.gnucash.org/wiki/De/Onlinebanking-Einrichtungsassistent#Schritt_f.C3.BCr_Schritt_Anleitung).
ISTR that 2 decades ago the automatic detection sometimes failed and it had to be set manually.

Reply:
OTOH a wrong HBCI version was the first detected cause of an error. later HTTP version, base64 encoding, … were added. So we should formulate  it more general here, perhaps:
Check the details of your connection like HBCI version and verify them in the log window..